docs/source/backends/openwrt.rst

```diff
@@ -2285,6 +2285,289 @@ Will be rendered as follows::
+
+Schema additions
+~~~~~~~~~~~~~~~~
+
+The ``OpenWrt`` backend adds a few properties to the WireGuard schema, see below.
+
++-----------------+---------+--------------+-------------------------------------------------------------+
+| key name        | type    | default      | description                                                 |
++=================+=========+==============+=============================================================+
+| ``network``     | string  | ``None``     | logical interface name (UCI specific),                      |
+|                 |         |              |                                                             |
+|                 |         |              | 2 to 15 alphanumeric characters, dashes and underscores     |
++-----------------+---------+--------------+-------------------------------------------------------------+
+| ``nohostroute`` | boolean | ``False``    | do not add routes to ensure the tunnel endpoints are routed |
+|                 |         |              | via non-tunnel device                                       |
++-----------------+---------+--------------+-------------------------------------------------------------+
+| ``fwmark``      | string  | ``None``     | firewall mark to apply to tunnel endpoint packets           |
++-----------------+---------+--------------+-------------------------------------------------------------+
+| ``ip6prefix``   | list    | ``[]``       | IPv6 prefixes to delegate to other interfaces               |
++-----------------+---------+--------------+-------------------------------------------------------------+
+| ``addresses``   | list    | ``[]``       | list of unique IPv4 or IPv6 addresses                       |
++-----------------+---------+--------------+-------------------------------------------------------------+
+
+The ``OpenWrt`` backend also adds ``wireguard_peers`` option for sepecifying a list of
+WireGuard Peers. It add the following properties to the ``wireguard_peers`` property of
+WireGuard schema.
+
++-----------------------+---------+-----------+-----------------------------------------------------------------+
+| key name              | type    | default   | description                                                     |
++=======================+=========+===========+=================================================================+
+| ``interface``         | string  | ``None``  | name of the wireguard interface,                                |
+|                       |         |           |                                                                 |
+|                       |         |           | 2 to 15 alphanumeric characters, dashes and underscores         |
++-----------------------+---------+-----------+-----------------------------------------------------------------+
+| ``route_allowed_ips`` | boolean | ``False`` | automatically create a route for each Allowed IPs for this peer |
```

Comment:
It's fine but `automatically create a route for each of the Allowed IPs for this peer` flows more smoothly.